### PR TITLE
fix: service wrapper was removed as it has been rolled into the Devic…

### PR DESCRIPTION
**Pull request by marcpfuller**
…eServiceSDK interface

Signed-off-by: Marc-Philippe Fuller <marc-philippe.fuller@intel.com>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rfid-llrp-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) N/A
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
 N/A

## Testing Instructions
make test

### CODE REVIEW

**Review by lenny-intel — CHANGES_REQUESTED** [review]
Nice job with the mocks!

File: internal/driver/discover_test.go [hunk 1]
```diff
@@ -224,7 +230,8 @@ func TestAutoDiscover(t *testing.T) {
 	params := makeParams()
 
 	// attempt to discover without emulator, expect none found
-	svc.clearDevices()
+	// svc.clearDevices()
```
[the remark this answers is not in the document]
Comment (marcpfuller) [reply internal/driver/discover_test.go:233]: I left them un commented,  as I wasnt sure if the way I did the mocks would be sufficient

File: internal/driver/driver.go [hunk 2]
```diff
@@ -142,10 +143,7 @@ func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *dsModels.As
 
 	d.asyncCh = asyncCh
 	d.deviceCh = deviceCh
-	d.svc = &DeviceSDKService{
-		DeviceService: service.RunningService(),
-		lc:            lc,
-	}
+	d.svc = service.RunningService()
```
[the remark this answers is not in the document]
Comment (marcpfuller) [reply internal/driver/driver.go:146]: I think either works. I wasnt sure  which should be used, so I used this instead

File: internal/driver/discover_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcpfuller) [reply internal/driver/discover_test.go:233]: I have removed the comment

File: internal/driver/discover_test.go [hunk 3]
```diff
@@ -260,14 +267,18 @@ func TestAutoDiscover(t *testing.T) {
...
 	emu.SetResponse(llrp.MsgGetReaderCapabilities, &readerCaps)
 
+	svc.On("GetDeviceByName", mock.Anything).Return(models.Device{}, fmt.Errorf("error")).Once()
 	discovered = autoDiscover(context.Background(), params)
 	if len(discovered) != 1 {
 		t.Fatalf("expected 1 discovered device, however got: %d", len(discovered))
 	}
-	svc.AddDiscoveredDevices(discovered)
+
+	// svc.AddDiscoveredDevices(discovered)
```
[the remark this answers is not in the document]
Comment (marcpfuller) [reply internal/driver/discover_test.go:276]: I have removed the comment

File: internal/driver/discover_test.go [hunk 4]
```diff
@@ -277,7 +288,7 @@ func TestAutoDiscover(t *testing.T) {
 		t.Errorf("error shutting down test emulator: %+v", err)
 	}
 	// reset
-	svc.clearDevices()
+	// svc.clearDevices()
```
[the remark this answers is not in the document]
Comment (marcpfuller) [reply internal/driver/discover_test.go:291]: I have removed the comment

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by lenny-intel — APPROVED** [review]

**Pull request merged**